src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -595,6 +664,12 @@ private class GroupData
             public readonly HubConnectionStore Connections = new HubConnectionStore();
         }
 
+        private class UserData
```

Comment:
Maybe instead of duplicating GroupData with a different name, rename GroupData and reuse. SubscriptionConnections?